### PR TITLE
CA-393194: Fix pvremove failure

### PR DESCRIPTION
**Pull request by stephenchengCloud**
In lvmohba test, the test failed at
`SR.destroy(sr_ref) -> removeVG -> pvremove ` with the lvm error message "No devices to process".
The root cause is that the PV which was to be removed
was not really used as PV when it comes to multipathed devices. 
The LVM official document says:
"Since each LUN has multiple device nodes in /dev  that point to the same underlying data,
they all contain the same LVM metadata
and thus LVM commands will find the same metadata
multiple times and report them as duplicates.
These duplicate messages are only warnings
and do not mean the LVM operation has failed.
Rather, they are alerting the user that only one of the devices has been used as a physical volume and the others are being ignored."  
Please refer to https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/duplicate_pv_multipath#duplicate_pv_multipath

This fix is to find the real PV in a VG before removing the VG.

Test passed: 4038532

### CODE REVIEW

File: drivers/lvutil.py [hunk 1]
```diff
@@ -529,6 +529,22 @@ def createVG(root, vgname):
...
+def getPVsInVG(vgname):
+    # Get PVs in a specific VG
+    pvs_ret = cmd_lvm([CMD_PVS, '--separator', ' ', '--noheadings', '-o', 'pv_name,vg_name'])
+    
+    # Parse each line to extract PV and VG information
+    # No need to handle exceptions here, return empty list if any error
+    pvs_in_vg = []
+    lines = pvs_ret.strip().split('\n')
+    for line in lines:
+        pv, vg = line.split()
```
Comment (LunfanZhang) [inline drivers/lvutil.py:541]: Can we guarantee that the output at _line:534_ consistently matches the pattern `/dev/pv_xxx vg_xxx ` in all scenarios? If not, here might be exceptions.

Comment (stephenchengCloud) [reply drivers/lvutil.py:541]: The '-o' option in the command at line 534 specifies the fields to display.

Comment (LunfanZhang) [reply drivers/lvutil.py:541]: I random selected a host and tested , and the output was: `/dev/sda3--noheadingsVG_XenStorage-6d5630c0-a016-b2b8-d5fa-e8faed85ca3a`

Comment (stephenchengCloud) [reply drivers/lvutil.py:541]: There is a space ' ' before --noheadings.

**Comment by MarkSymsCtx** [comment]
@rdn32 we probably want to backport this to 8.2 as I suspect it will resolve your partner issue.

**Review by MarkSymsCtx — APPROVED** [review]

**Review by TimSmithCtx — DISMISSED** [review]

**Comment by MarkSymsCtx** [comment]
Whilst this change is in itself fine we will need to add unit tests for the code modified in order to be able to release it. Either in this PR or in a follow-up.

**Comment by stephenchengCloud** [comment]
> Whilst this change is in itself fine we will need to add unit tests for the code modified in order to be able to release it. Either in this PR or in a follow-up.
> 
> test_lvutil.py has separate classes for the various operations and currently does not have anything for VG/PV management

Sure. I will add unit tests in this PR.

**Comment by stephenchengCloud** [comment]
Added unit tests. 
Passed:
drivers/lvutil.py                     511    260    134      9    47%

**Review by MarkSymsCtx — APPROVED** [review]

**Review by TimSmithCtx — APPROVED** [review]

**Pull request merged**